Add Basic Frame Support

hiporox opened the pull request:
Resolves most of #240.

Currently, these frames do no have keyboard support. But they render properly, and support button actions, including a prompt asking the user to make a full account instead of a readonly account when they click a button while not signed in.

I will work on adding hot key support for frames soon, but wanted to get something that adds the basic display functionality and gets to parity with Warcast first

hellno commented:
Thanks for working on this 🙌🏼 I've manually merged some of the work in this PR with the previous effort in #246. It was too much effort to merge both and clean up afterwards, so I went with 'first-come-first-serve' and used your improvements on the other branch 🫡 I hope that's a good solution for you

hiporox commented:
I'm totally fine with that. Reviewing the code right now that got merged in with #246 and it looks like some of the features I added got lost. This is largely the signer stuff that I added, including prompting the user to create a full account (if they are in read only mode) when they try to interact with a frame. I will pull the current version down and clean it up sometime tomorrow and make a new PR to add these features in
